[io] SelectNow_TcpIpConnection_TransferStressTest & SelectWithSelectorFix_TcpIpConnection_TransferStressTest & Select_TcpIpConnection_TransferStressTest

05:55:44   SelectNow_TcpIpConnection_TransferStressTest>TcpIpConnection_TransferStressBaseTest.testMediumPackets:76->TcpIpConnection_TransferStressBaseTest.testPackets:92->TcpIpConnection_TransferStressBaseTest.testPackets:121->HazelcastTestSupport.assertTrueEventually:1041 expected:<97017464> but was:<1>
05:55:44   SelectNow_TcpIpConnection_TransferStressTest>TcpIpConnection_TransferStressBaseTest.testSemiRealisticPackets:88->TcpIpConnection_TransferStressBaseTest.testPackets:92->TcpIpConnection_TransferStressBaseTest.testPackets:121->HazelcastTestSupport.assertTrueEventually:1041 expected:<138378768> but was:<1>
05:55:44   SelectNow_TcpIpConnection_TransferStressTest>TcpIpConnection_TransferStressBaseTest.testSmallPackets:70->TcpIpConnection_TransferStressBaseTest.testPackets:92->TcpIpConnection_TransferStressBaseTest.testPackets:121->HazelcastTestSupport.assertTrueEventually:1041 expected:<147853630> but was:<1>
05:55:44   SelectWithSelectorFix_TcpIpConnection_TransferStressTest>TcpIpConnection_TransferStressBaseTest.testMediumPackets:76->TcpIpConnection_TransferStressBaseTest.testPackets:92->TcpIpConnection_TransferStressBaseTest.testPackets:121->HazelcastTestSupport.assertTrueEventually:1041 expected:<205601416> but was:<1>
05:55:44   SelectWithSelectorFix_TcpIpConnection_TransferStressTest>TcpIpConnection_TransferStressBaseTest.testSemiRealisticPackets:88->TcpIpConnection_TransferStressBaseTest.testPackets:92->TcpIpConnection_TransferStressBaseTest.testPackets:121->HazelcastTestSupport.assertTrueEventually:1041 expected:<411205864> but was:<1>
05:55:44   SelectWithSelectorFix_TcpIpConnection_TransferStressTest>TcpIpConnection_TransferStressBaseTest.testSmallPackets:70->TcpIpConnection_TransferStressBaseTest.testPackets:92->TcpIpConnection_TransferStressBaseTest.testPackets:121->HazelcastTestSupport.assertTrueEventually:1041 expected:<496206730> but was:<1>
05:55:44   SelectWithSelectorFix_TcpIpConnection_TransferStressTest>TcpIpConnection_TransferStressBaseTest.testTinyPackets:64->TcpIpConnection_TransferStressBaseTest.testPackets:92->TcpIpConnection_TransferStressBaseTest.testPackets:121->HazelcastTestSupport.assertTrueEventually:1041 expected:<530200264> 
05:55:44   SelectNow_TcpIpConnection_TransferStressTest>TcpIpConnection_TransferStressBaseTest.testLargePackets:82->TcpIpConnection_TransferStressBaseTest.testPackets:121->HazelcastTestSupport.assertTrueEventually:1049->HazelcastTestSupport.sleepMillis:272 » TestTimedOut
05:55:44   SelectNow_TcpIpConnection_TransferStressTest>TcpIpConnection_TransferStressBaseTest.testTinyPackets:64->TcpIpConnection_TransferStressBaseTest.testPackets:92->TcpIpConnection_TransferStressBaseTest.testPackets:121->HazelcastTestSupport.assertTrueEventually:1049->HazelcastTestSupport.sleepMillis:272 » TestTimedOut
05:55:44   SelectWithSelectorFix_TcpIpConnection_TransferStressTest>TcpIpConnection_TransferStressBaseTest.testLargePackets:82->TcpIpConnection_TransferStressBaseTest.testPackets:121->HazelcastTestSupport.assertTrueEventually:1049->HazelcastTestSupport.sleepMillis:272 » TestTimedOut
05:55:44   Select_TcpIpConnection_TransferStressTest>TcpIpConnection_TransferStressBaseTest.testLargePackets:82->TcpIpConnection_TransferStressBaseTest.testPackets:121->HazelcastTestSupport.assertTrueEventually:1049->HazelcastTestSupport.sleepMillis:272 » TestTimedOut


https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-nightly/1358/consoleFull#1654123599befca796-5603-419b-b87e-3b231b9aafc7

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-nightly/1359/consoleFull#1654123599befca796-5603-419b-b87e-3b231b9aafc7

I ran those tests locally and I get `AssertionError`s like:
```java
java.lang.AssertionError: 
Expected :1059428668
Actual   :1
```
On Jenkins those tests seem to run into OOME, at least they are crashing the builds, e.g.:
https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-nightly/1374/consoleFull#1060638406255793cf-53c4-4118-98a6-ea4bf097b7a5

--- a/hazelcast/src/test/java/com/hazelcast/internal/networking/nio/SelectNow_TcpIpConnection_TransferStressTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/internal/networking/nio/SelectNow_TcpIpConnection_TransferStressTest.java
@@ -26,7 +26,6 @@ import org.junit.runner.RunWith;
 
 @RunWith(HazelcastSerialClassRunner.class)
 @Category(NightlyTest.class)
-@Ignore(value = "https://github.com/hazelcast/hazelcast/issues/10660")
 public class SelectNow_TcpIpConnection_TransferStressTest extends TcpIpConnection_TransferStressBaseTest {
 
     @Before

--- a/hazelcast/src/test/java/com/hazelcast/internal/networking/nio/SelectWithSelectorFix_TcpIpConnection_TransferStressTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/internal/networking/nio/SelectWithSelectorFix_TcpIpConnection_TransferStressTest.java
@@ -26,7 +26,6 @@ import org.junit.runner.RunWith;
 
 @RunWith(HazelcastSerialClassRunner.class)
 @Category(NightlyTest.class)
-@Ignore(value = "https://github.com/hazelcast/hazelcast/issues/10660")
 public class SelectWithSelectorFix_TcpIpConnection_TransferStressTest extends TcpIpConnection_TransferStressBaseTest {
 
     @Before

--- a/hazelcast/src/test/java/com/hazelcast/internal/networking/nio/Select_TcpIpConnection_TransferStressTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/internal/networking/nio/Select_TcpIpConnection_TransferStressTest.java
@@ -26,7 +26,6 @@ import org.junit.runner.RunWith;
 
 @RunWith(HazelcastSerialClassRunner.class)
 @Category(NightlyTest.class)
-@Ignore(value = "https://github.com/hazelcast/hazelcast/issues/10660")
 public class Select_TcpIpConnection_TransferStressTest extends TcpIpConnection_TransferStressBaseTest {
 
     @Before

--- a/hazelcast/src/test/java/com/hazelcast/internal/networking/spinning/Spinning_TcpIpConnection_TransferStressTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/internal/networking/spinning/Spinning_TcpIpConnection_TransferStressTest.java
@@ -26,7 +26,6 @@ import org.junit.runner.RunWith;
 
 @RunWith(HazelcastSerialClassRunner.class)
 @Category(NightlyTest.class)
-@Ignore(value = "https://github.com/hazelcast/hazelcast/issues/10660")
 public class Spinning_TcpIpConnection_TransferStressTest extends TcpIpConnection_TransferStressBaseTest {
 
     @Before

--- a/hazelcast/src/test/java/com/hazelcast/nio/tcp/TcpIpConnection_TransferStressBaseTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/nio/tcp/TcpIpConnection_TransferStressBaseTest.java
@@ -28,9 +28,9 @@ import org.junit.Before;
 import org.junit.Test;
 
 import java.util.Random;
-import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
 
+import static java.util.concurrent.TimeUnit.MINUTES;
 import static org.junit.Assert.assertEquals;
 
 /**
@@ -44,7 +44,7 @@ import static org.junit.Assert.assertEquals;
 public abstract class TcpIpConnection_TransferStressBaseTest extends TcpIpConnection_AbstractTest {
 
     // total running time for writer threads
-    private static final long WRITER_THREAD_RUNNING_TIME_IN_SECONDS = TimeUnit.MINUTES.toSeconds(2);
+    private static final long WRITER_THREAD_RUNNING_TIME_IN_SECONDS = MINUTES.toSeconds(2);
     // maximum number of pending packets
     private static final int maxPendingPacketCount = 10000;
     // we create the payloads up front and select randomly from them. This is the number of payloads we are creating
@@ -222,7 +222,8 @@ public abstract class TcpIpConnection_TransferStressBaseTest extends TcpIpConnec
                 } else {
                     normalPackets++;
                 }
-                //  writeHandler.write(packet);
+
+                c.getChannel().write(packet);
 
                 long now = System.currentTimeMillis();
                 if (now > prev + 2000) {

